stestr/config_file.py

```diff
@@ -33,19 +33,19 @@ def __init__(self, config_file):
         self.config_file = config_file
 
     def get_run_command(self, options, test_ids=None, regexes=None):
-        """Get a test_listing_fixture.TestListingFixture for this config file
+        """Get a test_processor/.TestProcessorFixture for this config file
```

Comment:
Nope that was a mistake, I'll fix it in a new revision. I'm going to wait until after #66 merges before I bother though.